lib/generators/ember/bootstrap_generator.rb

```diff
@@ -68,6 +60,17 @@ def inject_into_application_file(safe_extension)
           template application_file, full_path
         end
       end
+
+      def get_options_from_path(path)
```

Comment:
@tricknotes Do you mean that the method should be named `get_options_from_contents`? Or that the parameter should be named `contents`?